flink-runtime-web/web-dashboard/app/scripts/modules/submit/submit.ctrl.coffee

```diff
@@ -167,7 +167,7 @@ angular.module('flinkApp')
             $scope.uploader['success'] = null
           else
             $scope.uploader['success'] = "Uploaded!"
-      xhr.open("POST", "/jars/upload")
+      xhr.open("POST", "jars/upload")
```

Comment:
Can you use `flinkConfig.jobServer + "jars/upload"` here? Otherwise, looks good to me.